static/src/javascripts/projects/commercial/modules/creatives/page-skin.js

```diff
@@ -40,19 +41,66 @@ const pageSkin = (): void => {
         }
     };
 
-    const initTopPositionOnce = (): void => {
+    const initTopPositionOnce = (hasTruskin: boolean): void => {
         if (topPosition === 0) {
             const navHeader = document.getElementsByClassName('new-header')[0];
             if (navHeader) {
-                topPosition = navHeader.offsetTop + navHeader.offsetHeight;
+                topPosition = hasTruskin
+                    ? navHeader.offsetTop + adLabelHeight
+                    : navHeader.offsetTop + navHeader.offsetHeight;
             }
         }
     };
 
-    // This is to reposition the Page Skin to start where the navigation header ends.
+    const shrinkElement = (element: HTMLElement | null): void => {
+        const frontContainer = document.querySelector('.fc-container__inner');
+        if (element && frontContainer) {
+            element.style.cssText = `max-width: ${
+                frontContainer.clientWidth
+            }px; margin-right: auto; margin-left: auto;`;
+        }
+    };
+
     const repositionSkin = (): void => {
-        if (hasPageSkin && isInAUEdition) {
-            initTopPositionOnce();
+        const bodyElement = document.querySelector('body');
+        const hasTruskin = bodyElement
+            ? bodyElement.classList.contains('truskin-page-skin')
+            : false;
+        const header = document.querySelector('.new-header');
+        const footer = document.querySelector('.l-footer');
+        const topBannerAd = document.querySelector('.ad-slot--top-banner-ad');
+
+        if (hasTruskin && header && topBannerAd) {
+            initTopPositionOnce(hasTruskin);
+
+            shrinkElement(header);
+            shrinkElement(footer);
+
+            if (window.pageYOffset === 0) {
+                moveBackgroundVerticalPosition(topPosition);
+            }
+
+            const headerBoundaries = header.getBoundingClientRect();
+            const topBannerAdBoundaries = topBannerAd.getBoundingClientRect();
+            const headerPosition = headerBoundaries.top;
+            const fabricScrollStartPosition =
+                topBannerAdBoundaries.height +
+                adLabelHeight -
+                headerBoundaries.height;
+
+            if (
+                headerPosition <= fabricScrollStartPosition &&
+                topBannerAdBoundaries.bottom > 0
```

Comment:
We're looking up `topBannerAdBoundaries.bottom` three times in this `if` block, it could be assigned to a variable as done with `headerPosition`.